envs/monkey_zoo/blackbox/analyzers/analyzer_log.py

```diff
@@ -0,0 +1,17 @@
+LOG_INIT_MESSAGE = "Analysis didn't run."
+
+
+class AnalyzerLog(object):
```

Comment:
Why develop a log and not use `logging`?

Reply:
Because instead of displaying all messages, I want to just keep N last messages and display only them. Maybe this could be done with logging, but I couldn't find it.